Improve documentation for publishedService and IP options (#9380)
traefik/traefik: docs/content/providers/kubernetes-ingress.md

_Optional, Default: ""_

> This IP will get copied to ingress status.loadbalancer.ip and currently only supports one IPv4 or IPv6 address.

Reviewer: ```suggestion
This IP will get copied to Ingress status.loadbalancer.ip and currently only supports one IP value (IPv4 or IPv6).
```

> This IP will get copied to Ingress status.loadbalancer.ip and currently only supports one IP value (IPv4 or IPv6).

Reviewer 2: ```suggestion
This IP will get copied to Ingress `status.loadbalancer.ip` and currently only supports one IP value (IPv4 or IPv6).
```

> This IP will get copied to Ingress `status.loadbalancer.ip` and currently only supports one IP value (IPv4 or IPv6).

Reviewer 3: ```suggestion
This IP will get copied to Ingress `status.loadbalancer.ip`, and currently only supports one IP value (IPv4 or IPv6).
```

_Optional, Default: ""_

The Kubernetes service to copy status from.

> When using third parties tools like External-DNS, this option can be used to copy the service loadbalancer.status (containing the service's endpoints IPs) to the ingresses.

Reviewer 2: ```suggestion
When using third parties tools like External-DNS, this option can be used to copy the service `loadbalancer.status` (containing the service's endpoints IPs) to the ingresses.
```

_Optional, Default: ""_

> Published Kubernetes Service to copy status from.
> It will use the services IPs to copy to ingress loadbalancer status, useful for example External-DNS.

Reviewer: ```suggestion
The Kubernetes service to copy status from.
When using third parties tools like External-DNS, this option can be used to copy the service loadbalancer.status (containing the service's endpoints IPs) to the ingresses.
```
